scripts/update/Updater.php

```diff
@@ -1234,6 +1234,6 @@ public function update($initialVersion) {
             $this->setVersion('9.3.3');
         }
 
-        $this->skip('9.3.3', '9.4.1');
+        $this->skip('9.3.3', '9.5.0');
```

Comment:
You must also provide an update script to set the default value of the new option. Otherwise the existing behavior of the progress bar will be lost.